pkg/pubsub/operations/topic.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2019 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package operations
```

Comment:
I think you need a short README.md in `pkg/pubsub/operations/` explaining the model and usage of these jobs. (Something similar to the description I overheard in person would be helpful.)

Reply:
Good idea. Will do.

Reply:
Added.